examples/99-advanced/anatomical_groups.py

```diff
@@ -56,14 +56,10 @@
 #     This mapping is specific to the PyVista datasets and is not part of the
 #     TotalSegmentator data.
 
-print(
-    '{\n'
-    + '\n'.join(
-        f'    {"'" + name + "':":<32} ({R:>3}, {G:>3}, {B:>3}),'
-        for name, (R, G, B) in names_to_colors.items()
-    )
-    + '\n}',
-)
+print('{')
+for name, (R, G, B) in names_to_colors.items():
+    print(f'{repr(name):<32}: ({R:>3}, {G:>3}, {B:>3}),')
```

Comment:
Sorry, I brainfarted my comment from earlier. I meant
```suggestion
    print(f'{name!r:<32}: ({R:>3}, {G:>3}, {B:>3}),')
```
This is what this format specifier was made for. But if you think it's more obscure than explicit repr than I don't insist.

Reply:
That indeed feels a little obscure to me

Reply:
The important thing here for me is that the printed dict looks good. I wanted users to be able to easily see and copy the color mapping without needing to download the dataset. So from that perspective, I'm indifferent to how this code is formatted. Feel free to revert to use `repr`.

Reply:
Ag didn't realize it was committed, this works for me as just having this in a traditional for loop makes it readable enough

Reply:
The suggestion by @adeak is also consistent with [RUF010](https://docs.astral.sh/ruff/rules/explicit-f-string-type-conversion/), which we have enabled. Pre-commit was failing previously when using `repr`:

https://results.pre-commit.ci/run/github/92974124/1741124058.AyC2Vz0oRcW7U2xvhkSxQA

So I think this is the right choice.